src/main/java/bio/terra/service/snapshot/SnapshotDao.java

```diff
@@ -279,6 +304,20 @@ private Snapshot retrieveWorker(String sql, MapSqlParameterSource params) {
         }
     }
 
+    private SnapshotProjectModel retrieveSnapshotProject(String sql, MapSqlParameterSource params) {
+        try {
+            SnapshotProjectModel snapshotProject = jdbcTemplate.queryForObject(sql, params, (rs, rowNum) ->
+                new SnapshotProjectModel()
+                    .id(rs.getString("id"))
+                    .name(rs.getString("name"))
+                    .dataProject(rs.getString("dataProject"))
```

Comment:
I think using mixed case may fail since the jdbc driver for postgres represents the fields as lower case unless you explicitly define or select them in double quotes...Worth double checking.